src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -162,6 +162,11 @@ - (void)cleanup
         delete _pairingDelegateBridge;
         _pairingDelegateBridge = nullptr;
     }
+
+    if (_partialDACVerifier) {
```

Comment:
This looks unrelated to https://github.com/project-chip/connectedhomeip/issues/22197

Reply:
This is addressing the issue in https://github.com/project-chip/connectedhomeip/pull/22115#pullrequestreview-1087634372